Sources/Architecture/Coordinator.swift

```diff
@@ -42,6 +42,12 @@ public protocol ModalCoordinator: DefaultCoordinator {
     weak var destinationNavigationController: UINavigationController? { get }
 }
 
+public protocol TabBarCoordinator: DefaultCoordinator {
```

Comment:
Nebyl by správnější název `TabBarItemCoordinator`? Chápu to tak, že nespravuje samotný tab bar, ale jen jeho položku (tab).

Reply:
To je docela dobrý nápad.

Reply:
A pak můžeme mít i opravdový TabBarCoordinator (nebo alespoň template na něj), který bude brát Window.